src/test/unit/test_main_utils.py

```diff
@@ -194,27 +197,57 @@ def func():
     assert frames[0] == {
         "function": "func",
         "fileName": __file__,
-        "variables": {"a": "A"},
+        "variables": {"a": '"A"'},
         "lineno": frames[1]["lineno"] - 3,
     }
 
 
 @pytest.mark.parametrize(
     ("value", "output"),
     [
-        ("aa", "aa"),  # happy flow
-        (None, "None"),  # same key twice
-        ("a" * 21, "a" * 20 + "...[too long]"),
-        ({"a": "a"}, '{"a": "a"}'),  # dict.
-        # dict that can't be converted to json.
-        ({"a": set()}, "{'a': set()}"),  # type: ignore
-        (b"a", "a"),  # bytes that can be decoded.
-        (b"\xff\xfea\x00", "b'\\xff\\xfea\\x00'"),  # bytes that can't be decoded.
-        ({1: Decimal(1)}, '{"1": 1.0}'),  # decimal should be serializeable
+        ("aa", '"aa"'),  # happy flow - string
+        (None, "null"),  # happy flow - None
+        ("a" * 101, '"' + "a" * 99 + "...[too long]"),  # simple long string
+        ({"a": "a"}, '{"a": "a"}'),  # happy flow - dict
+        ({"a": set()}, '{"a": "set()"}'),  # dict that can't be converted to json
```

Comment:
?
Think you need to solve this

Reply:
that's on purpose 😂 
what should we do if the object is not serializable?